openreview/venue/invitation.py

```diff
@@ -1213,6 +1213,35 @@ def set_bid_invitations(self):
 
             bid_invitation = self.save_invitation(bid_invitation, replacement=True)
 
+            configuration_invitation = tools.get_invitation(self.client, f'{match_group_id}/-/Assignment_Configuration')
+            if configuration_invitation:
+                scores_spec = configuration_invitation.edit['note']['content']['scores_specification']
+                if bid_invitation.id not in scores_spec['value']['param']['default']:
+                    scores_spec['value']['param']['default'][bid_invitation.id] = {
+                        'weight': 1,
+                        'default': 0,
+                        'translate_map' : {
```

Comment:
The changes look good, I think we should store this mapping function somewhere so it is not repeated in the code. 

What about storing it in the Bid Invitation content and then read it to set the score spec?